recipes/quetz-sql-authenticator/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "quetz-sql-authenticator" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/quetz-sql-authenticator-{{ version }}.tar.gz
+  sha256: 49212cf94c664723df9c7d24b1f44e87b06daa6f1522692095f6ceef7e6f8a6d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - quetz >=0.6.1
+    - sqlalchemy
+    - fastapi
+    - alembic
+    - passlib
+
+test:
+  imports:
+    - quetz_sql_authenticator
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mamba-org/quetz-sql-authenticator
```

Comment:
Are you aware this URL gives a 404?

Reply:
Yes, we're gonna make it public soon 👍🏼 Is that required to create the feedstock?

Reply:
No, it's not required, it just generally makes it easier to review by being able to click through the repository rather than downloading the sdist from PyPI.

It's also not at all clear to me what the benefit is for having a private repo when the code is already up on PyPI.

Reply:
I don't have the rights to make it public, but it's in the works.